<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Example_bacterial_genome</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
  <link rel="stylesheet" href="Example_bacterial_genome/.styles.css" />
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Example_bacterial_genome</h1>
</header>
<h1>
Auckland Genomics Analysis Report for Example_bacterial_genome
</h1>
<p><img src="../images/Auckland_Genomics_Logo_1.png" alt="Logo" width="250"></p>
<h2 id="table-of-contents">Table of Contents</h2>
<ul>
<li><a href="#nanoplot-reports">NanoPlot Reports</a></li>
<li><a href="#data-stats">Data Stats</a></li>
<li><a href="#polished-assembly">Polished Assembly</a></li>
<li><a href="#bandage-visualization">Bandage Visualization</a></li>
<li><a href="#annotation">Annotation</a></li>
<li><a href="#abricate-output-toxins-and-amr-genes">Abricate Output (toxins and AMR genes)</a></li>
<li><a href="#assembly-coverage-stats">Assembly Coverage Stats</a></li>
<li><a href="#taxonomic-assignment">Taxonomic Assignment</a></li>
<li><a href="#possible-polymorphism-html-report">Possible Polymorphism HTML Report</a></li>
<li><a href="#kraken-report-unmapped-reads">Kraken Report (unmapped reads)</a></li>
<li><a href="#krona-plot-unmapped-reads">Krona Plot (unmapped reads)</a></li>
<li><a href="#multiqc-report">MultiQC Report</a></li>
<li><a href="#program-versions">Program Versions</a></li>
</ul>
<h2 id="nanoplot-reports">NanoPlot Reports</h2>
<p>Quality assessment and statistical analysis of raw and trimmed reads, using NanoPlot. Includes both log and non-log plots. The logged read length and quality of the raw reads is below <img src="results/Example_bacterial_genome_NanoPlot_report_raw/LengthvsQualityScatterPlot_loglength_kde.png" alt="Nanoplot" width="300"></p>
<p><a href="results/Example_bacterial_genome_NanoPlot_report_raw/NanoPlot-report.html">Full NanoPlot Report Raw Data</a></p>
<p><a href="results/Example_bacterial_genome_NanoPlot_report_trimmed/NanoPlot-report.html">Full NanoPlot Report Trimmed Data</a></p>
<h2 id="data-stats">Data Stats</h2>
<p>Summary of original and trimmed read sets (used for assembly). If the data consists of short fragmented reads, then the assembly will not continue as the long read assembler Flye will fail:</p>
<table>
<thead>
<tr class="header">
<th>File Path</th>
<th># Sequences</th>
<th>Total Length</th>
<th>Min Length</th>
<th>Avg Length</th>
<th>Max Length</th>
<th>Median</th>
<th>N50</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Example_bacterial_genome.fastq</td>
<td>118238</td>
<td>191412593</td>
<td>5</td>
<td>1618.9</td>
<td>193350</td>
<td>550.0</td>
<td>4330</td>
</tr>
<tr class="even">
<td>trimmed.fastq</td>
<td>44433</td>
<td>165663912</td>
<td>1000</td>
<td>3728.4</td>
<td>193350</td>
<td>2451.0</td>
<td>5233</td>
</tr>
</tbody>
</table>
<h2 id="polished-assembly">Polished Assembly</h2>
<p><a href="results/Example_bacterial_genome_consensus.fasta">Polished Assembly Fasta File</a></p>
<h2 id="bandage-visualization">Bandage Visualization</h2>
<p>Graphical representation of the assembly graph, generated by Bandage:</p>
<p><img src="results/Example_bacterial_genome_bandage.png" alt="Bandage Visualization" width="300"></p>
<h2 id="annotation">Annotation</h2>
<p>Functional annotation of the polished assembly using <strong>Prokka</strong>. This is your annotated genome. If you open the .gbk (GenBank) file in a tool like Geneious, you will see the entire genome with each gene labelled.</p>
<p><a href="results/Example_bacterial_genome_prokka/">Annotation Folder</a></p>
<h2 id="abricate-output-toxins-and-amr-genes">Abricate Output (toxins and AMR genes)</h2>
<p>Identification by <a href="https://github.com/tseemann/abricate">Abricate</a> of potential toxins and antimicrobial resistance (AMR) genes:</p>
<table>
<thead>
<tr class="header">
<th>SEQUENCE</th>
<th>START</th>
<th>END</th>
<th>STRAND</th>
<th>GENE</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>contig_1</td>
<td>177791</td>
<td>178969</td>
<td>-</td>
<td>cmlR</td>
</tr>
<tr class="even">
<td>contig_1</td>
<td>321320</td>
<td>322413</td>
<td>-</td>
<td>cmlV</td>
</tr>
<tr class="odd">
<td>contig_1</td>
<td>1908361</td>
<td>1909890</td>
<td>-</td>
<td>fusH</td>
</tr>
<tr class="even">
<td>contig_1</td>
<td>1929871</td>
<td>1931127</td>
<td>-</td>
<td>mgt</td>
</tr>
<tr class="odd">
<td>contig_1</td>
<td>1931124</td>
<td>1931906</td>
<td>-</td>
<td>erm(O)</td>
</tr>
<tr class="even">
<td>contig_1</td>
<td>4400605</td>
<td>4401381</td>
<td>-</td>
<td>aac(3)-IVa</td>
</tr>
<tr class="odd">
<td>contig_1</td>
<td>4689308</td>
<td>4689916</td>
<td>-</td>
<td>vanX-Sc</td>
</tr>
<tr class="even">
<td>contig_1</td>
<td>4689913</td>
<td>4690953</td>
<td>-</td>
<td>vanA-Sc</td>
</tr>
<tr class="odd">
<td>contig_1</td>
<td>4690964</td>
<td>4691977</td>
<td>-</td>
<td>vanH-Sc</td>
</tr>
<tr class="even">
<td>contig_1</td>
<td>4692072</td>
<td>4693265</td>
<td>-</td>
<td>vanK-Sc</td>
</tr>
<tr class="odd">
<td>contig_1</td>
<td>4693456</td>
<td>4694448</td>
<td>-</td>
<td>vanJ</td>
</tr>
<tr class="even">
<td>contig_1</td>
<td>4694652</td>
<td>4695348</td>
<td>+</td>
<td>vanR-O</td>
</tr>
<tr class="odd">
<td>contig_1</td>
<td>7752739</td>
<td>7754646</td>
<td>-</td>
<td>otr(A)</td>
</tr>
</tbody>
</table>
<h2 id="coverage-stats">Coverage Stats</h2>
<table>
<thead>
<tr class="header">
<th>Chromosome</th>
<th>Length</th>
<th>Bases</th>
<th>Mean Coverage</th>
<th>Min Coverage</th>
<th>Max Coverage</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>contig_1</td>
<td>8571456</td>
<td>172264861</td>
<td>20.10</td>
<td>1</td>
<td>86</td>
</tr>
<tr class="even">
<td>total</td>
<td>8571456</td>
<td>172264861</td>
<td>20.10</td>
<td>0</td>
<td>86</td>
</tr>
</tbody>
</table>
<h2 id="taxonomic-assignment">Taxonomic Assignment</h2>
<p>When the taxon is known, this is indicated. Otherwise, they are left as unknown.</p>
<p>Domain Bacteria<br />
Phylum Actinobacteria<br />
Class Actinobacteria<br />
Order Streptomycetales<br />
Family Streptomycetaceae<br />
Genus Streptomyces<br />
Species unknown<br />
Strain unknown</p>
<h2 id="possible-polymorphism-html-report">Possible Polymorphism HTML Report</h2>
<p>Report on possible genetic polymorphisms in the assembly. It is very likely some of these calls are due to misassembly, so care should be taken in interpretation. They may also truly indicate contamination or non-clonality. It is <strong>strongly</strong> recommended that regions in which polymorphisms are inferred are checked for coverage by generating files with <a href="https://github.com/lh3/minimap2">minimap2</a> and visually examining in Geneious or <a href="https://igv.org/">IGV</a>. Unexpected variation in coverage may indicate assembly errors. This output is generated using the –polymorphism mode in <a href="https://barricklab.org/twiki/bin/view/Lab/ToolsBacterialGenomeResequencing">breseq</a>:</p>
<p><a href="results/Example_bacterial_genome_assembly_polymorphism_evidence/output/index.html">Possible Polymorphism HTML Report</a></p>
<h2 id="kraken-report-unmapped-reads">Kraken Report (unmapped reads)</h2>
<p>After assembly, raw reads are mapped back to the assembly. Many of these do not map (and were not used in the assembly). To understand the source of extraneous reads, taxonomic classification of them is done using <a href="https://github.com/DerrickWood/kraken2">Kraken2</a>. Often these reads are human contamination, but they can also be from other contaminants (e.g. other bacterial strains). Columns in the report are:</p>
<ol type="1">
<li><p>% reads assigned to a taxon and all subtaxa;</p></li>
<li><p>Number of reads assigned to that taxon and all subtaxa;</p></li>
<li><p>Number of reads assigned to that taxon (excluding subtaxa);</p></li>
<li><p>Rank (‘P’ phylum, ‘C’ class, ‘O’ order, ‘F’ family, ‘G’ genus, ‘S’ species);</p></li>
<li><pre><code>    NCBI Taxonomy ID;</code></pre></li>
<li><p>Scientific name.</p></li>
</ol>
<p><a href="results/Example_bacterial_genome_kraken2_report_unmapped.txt">Kraken Report</a></p>
<h2 id="krona-plot-unmapped-reads">Krona Plot (unmapped reads)</h2>
<p>This is an interactive visualization of the Kraken2-classified unmapped reads:</p>
<p><a href="results/Example_bacterial_genome_krona_report.html">Krona Interactive HTML</a></p>
<h2 id="multiqc-report">MultiQC Report</h2>
<p>Summary report aggregating results from multiple bioinformatics tools including depth (MosDepth), Samtools flagstat, and Kraken2 brief summary:</p>
<p><a href="results/Example_bacterial_genome_multiqc_report.html">MultiQC Report</a></p>
<h2 id="program-versions">Program Versions</h2>
<p>Software versions used during the analysis:</p>
<p>Snakemake: 7.32.3</p>
<p>Python: 3.11.6</p>
<p>Filtlong: v0.2.0</p>
<p>Seqkit: v2.4.0</p>
<p>Flye: 2.9.3-b1797</p>
<p>Bandage: 0.8.1</p>
<p>Medaka: 1.11.1</p>
<p>Prokka: 1.14.5</p>
<p>Breseq: 0.38.3</p>
<p>Minimap2: 2.26-r1175</p>
<p>Kraken2: 2.1.3</p>
<p>Mosdepth: 0.3.4</p>
<p>R: 4.3.2</p>
<p>Perl: 5.34.1</p>
<hr />
<p>Report generated by <a href="https://www.auckland.ac.nz/en/science/about-the-faculty/share-shared-research-equipment/auckland-genomics.html">Auckland Genomics</a> on Mon May 19 17:56:49 NZST 2025</p>
<p><em>Empowering Breakthroughs Through Genomics</em></p>
<script>
document.addEventListener("DOMContentLoaded", function() {
    // Select all anchor tags
    var links = document.querySelectorAll('a[href$=".pdf"]');
    
    // Loop through all PDF links and add target="_blank"
    links.forEach(function(link) {
        link.setAttribute('target', '_blank');
    });
});
</script>
</body>
</html>
